examples/MaxHeightExample.js

```diff
@@ -13,23 +13,29 @@ class MaxHeightExample extends React.Component {
   constructor(props) {
     super(props);
 
-    this.addRow = this.addRow.bind(this);
     this.state = {
       dataList: new FakeObjectDataListStore(5),
     };
   }
 
-  addRow() {
-    this.setState({
-      dataList: new FakeObjectDataListStore(this.state.dataList.getSize() + 1),
-    });
-  }
+  addRow = () => {
+    this.state.dataList.setSize(this.state.dataList.getSize() + 1);
```

Comment:
does this also create a new data object for the newly added row?

Reply:
nvm just saw changes in fakeobjectdataliststore file

Reply:
Yea the actual row entries are only created on demand.